<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PLMSettings</class>
 <widget class="QWidget" name="PLMSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="hLayout_PLMServerURL">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelURL_8">
       <property name="text">
        <string>PLM Server URL</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_PLMServerURL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="hLayoutProtocol">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelURL_9">
       <property name="text">
        <string>Protocol</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Protocol">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>https</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>http</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ssh</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="hLayout_EndPoint">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelURL_6">
       <property name="text">
        <string>End Point</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_EndPoint">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="hLayout_ConfigurationFile">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelURL_5">
       <property name="text">
        <string>Configuration File</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_ConfigurationFile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>262</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>270</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbOpen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>19</width>
         <height>19</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>19</width>
         <height>19</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Resources/btn_icon_open_none.svg</normaloff>:/Resources/btn_icon_open_none.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbRefresh">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>19</width>
         <height>19</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>19</width>
         <height>19</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Resources/btn_icon_refresh_none.svg</normaloff>:/Resources/btn_icon_refresh_none.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
